code/drasil-printers/lib/Language/Drasil/Markdown/CreateMd.hs

```diff
@@ -96,24 +97,15 @@ addListToTuple _ _ = []
 license :: Doc -> Doc
 license auth = text "Copyright (c) 2021," <+> auth <>
   text ". All rights reserved. Please see the [full license](https://github.com/JacquesCarette/Drasil/blob/4b9ad0a3016fecb3c7a2aa82ab142f9e805b5cc8/LICENSE) for more details."
--- | Drasil Tree icon. Uses HTML directly to format image since normal markdown doesn't support it.
-drasilImage :: Doc
-drasilImage = alignImage "../../../../drasil-website/WebInfo/images/Icon.png"
--- | Aligns an image to the center using HTML, since markdown doesn't support it.
-alignImage :: FilePath -> Doc
-alignImage img = text "<p align=\"center\">" <>
-  contSep <> text ("<img src=\"" ++ img ++ "\" alt=\"Drasil Tree\" width=\"200\" />")
-  <> contSep <> text "</p>"
+
 -- | End section.
-endNote :: [String] -> Doc
-endNote auth = text "*This README is a software artifact generated by Drasil.*" <> doubleSep <> license (listToDoc auth) <> doubleSep <> drasilImage 
+endNote :: Int -> [String] -> Doc
+endNote num auth = text "*This README is a software artifact generated by Drasil.*" <> doubleSep <> license (listToDoc auth) <> doubleSep <> drasilImage num
 
 -- | Section seperators.
-secSep, contSep, doubleSep, bkQuote, bkQuote3 :: Seperator
+secSep, doubleSep, bkQuote, bkQuote3 :: Seperator
```

Comment:
How much of a change would it be to change `Seperator` to `Separator` to fix the typo throughout the code/comments? Should it be a separate PR or is it simple enough to do it here?

Reply:
It shouldn't be much work. It just needs to be fixed in 4 places I believe so I think we can do it in this PR itself.